plugins/polio/tasks/vaccine_authorizations_mail_alerts.py

```diff
@@ -83,6 +86,11 @@ def send_email_vaccine_authorizations_60_days_expiration_alert(task=None):
     email_sent = 0
 
     for i, vacc_auth in enumerate(vaccine_auths):
+        try:
+            gpei_coordinator = Profile.objects.get(user__username__istartswith="gpei", org_units=vacc_auth.country)
+            mailing_list.append(gpei_coordinator.user.email)
```

Comment:
There is a GPEI coordinator field in the campaign object. It seems more reliable to me that filtering by user name. Or we could add a field to set the GPEI coordinator in the polio country config